test/fail_compilation/test17451.d

```diff
@@ -40,6 +40,6 @@ struct ThreadSlot {
         ArraySet!Task tasks;
 }
 
-class Libevent2ManualEvent {
+class Libevent2ManualEvent : ManualEvent {
```

Comment:
I temporarily reverted the fix for this issue to verify that it still occurred with this change.